src/cfnlint/runner.py

```diff
@@ -49,7 +49,9 @@ def run(self):
             matches.extend(
                 self.rules.run(
                     self.filename, self.cfn))
+        return self.check_directives(matches)

```

Comment:
what is a directive?? This function deserves a rename

Reply:
Will refactor!

Reply:
Changed in new patch